src/internal/observable/timer.ts

```diff
@@ -38,14 +38,14 @@ import { Subscriber } from '../Subscriber';
  * @see {@link index/interval}
  * @see {@link delay}
  *
- * @param {number|Date} [dueTime] The initial delay time to wait before
- * emitting the first value of `0`.
+ * @param {number|Date} [dueTime] The initial delay time specified as a Date object or as an integer denoting
+      milliseconds between emissions to wait before emitting the first value of 0`.
```

Comment:
I think "between emissions" should be removed, here.

Reply:
yep I think so too.

Reply:
BTW, congratulations on joining the core team :tada:

Reply:
Thank you very much. I really appreciate this :smile: